Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/duffelbag.yml

```diff
@@ -19,3 +19,16 @@
     contents:
     - id: BoxSurvivalMedical
     - id: RubberStampPsychologist
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackDuffelCaptain
+  id: ClothingBackpackDuffelAdminAssistantFilled
+  name: administrative assistant's duffel bag
+  description: Its a duffel bag build for carrying paper.
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvival
+      - id: Flash
```

Comment:
Duffelbag and Satchel still have a flash, which I believe you intended to remove. Only the backpack has the flash removed